audit-drivers.md

```diff
@@ -58,3 +58,36 @@ class MyCustomDriver implements AuditDriver
 ```
 
 > {tip} The `Database` driver is a good starting point to get ideas for a new custom driver implementation.
+
+## Defining the custom driver
+
+Here's how to set the custom driver `MyCustomDriver`:
+```php
+<?php
+namespace App\Models;
+
+use App\AuditDrivers\MyCustomDriver;
+use Jenssegers\Mongodb\Eloquent\Model;
```

Comment:
Replace with `use Illuminate\Database\Eloquent\Model;`, to avoid issues with copy/paste and users not realising this is for MongoDB.